cli/printer/printer.go

```diff
@@ -52,6 +54,10 @@ var DefaultPrinter = Printer{
 	},
 }
 
+func (p *Printer) SetSchema(schema *llx.Schema) {
```

Comment:
given llx.Schema is an interface, do we need this to be a pointer?

Reply:
I can't remember why I implemented it this way. It also works without a pointer and the code is nicer to read.

Thanks for the hint. I removed the pointer.